Source/control.cpp

```diff
@@ -1675,33 +1675,33 @@ int CapStatPointsToAdd(int remainingStatPoints, const PlayerStruct &player, attr
 void ReleaseChrBtns(bool addAllStatPoints)
 {
 	chrbtnactive = false;
-	for (int i = ATTRIB_STR; i <= ATTRIB_VIT; ++i) {
-		if (!chrbtn[i])
+	for (auto i : enum_values<CharacterAttribute>()) {
+		if (!chrbtn[static_cast<size_t>(i)])
 			continue;
 
-		chrbtn[i] = false;
-		if (MouseX >= ChrBtnsRect[i].x
-			&& MouseX <= ChrBtnsRect[i].x + ChrBtnsRect[i].w
-			&& MouseY >= ChrBtnsRect[i].y
-			&& MouseY <= ChrBtnsRect[i].y + ChrBtnsRect[i].h) {
+		chrbtn[static_cast<size_t>(i)] = false;
+		if (MouseX >= ChrBtnsRect[static_cast<size_t>(i)].x
+			&& MouseX <= ChrBtnsRect[static_cast<size_t>(i)].x + ChrBtnsRect[static_cast<size_t>(i)].w
+			&& MouseY >= ChrBtnsRect[static_cast<size_t>(i)].y
+			&& MouseY <= ChrBtnsRect[static_cast<size_t>(i)].y + ChrBtnsRect[static_cast<size_t>(i)].h) {
 			PlayerStruct &player = plr[myplr];
 			int statPointsToAdd = 1;
 			if (addAllStatPoints)
-				statPointsToAdd = CapStatPointsToAdd(player._pStatPts, player, (attribute_id)i);
+				statPointsToAdd = CapStatPointsToAdd(player._pStatPts, player, (CharacterAttribute)i);

```

Comment:
It's `i` actually a `CharacterAttribute` type now, so this cast is now redundant?

Reply:
Yep, fixed.